steroids/src/app/exercice/exercice.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, OnInit, Input } from '@angular/core';
+import Exercice from 'src/model/Exercice';
+import { PersistanceService } from '../persistance.service';
+import Series from 'src/model/Series';
+
+@Component({
+  selector: 'app-exercice',
+  templateUrl: './exercice.component.html',
+  styleUrls: ['./exercice.component.css']
+})
+export class ExerciceComponent implements OnInit {
+
+  @Input()
+  exercice: Exercice;
+
+  serietoEdit: Series;
+
+  constructor(private persistance: PersistanceService) {
+    this.persistance.sessionsChanged.subscribe(_ => this.updateExercice());
+  }
+
+  ngOnInit() {
+  }
+
+  getSerietoEdit(serie: Series) {
+    this.serietoEdit = serie;
+  }
+
+  delete(idSerie: string) {
+    if(confirm("Are you sure to delete this serie ?")) {
```

Comment:
Sometimes you use single quote and sometimes double quote. You should stick to only one.